documentation/modules/additional-auth-methods-ostack.adoc

```diff
@@ -0,0 +1,9 @@
+[id="additional-authentication-methods-osp_{context}"]
+= Additional authentication methods for migrations with {osp} source providers
+
+{project-short} versions 2.6 and later support the following authentication methods for migrations with {osp} source providers in addition to the standard username and password credential set:
+
+* Token authentication
+* Application credential authentication
+
+You can use these methods to migrate virtual machines with {osp} source providers using the CLI the same way you migrate other virtual machines, except for how you prepare the `Secret` manifest.
```

Comment:
"Use these methods..."   I'm not quite sure what "except how you prepare the Secret manifest" is saying.  Does this mean I should not use these methods to prepare the secret manifest?  If so, I'd tell them not to use the methods, and do we need to tell them how they prepare the manifest in this case?

Reply:
"but do not use these methods to prepare the secret manifest."   (?)